documents/knowledge/Elements.txt

```diff
@@ -659,11 +659,11 @@ d (Double)
 -------------------------------
 e (Exponentiation)
 
-- Exponentiate two numbers / get every nth character of a string / get length of a regex match
+- Exponentiate two numbers / extend string / get length of a regex match
 
     num a, num b: a ** b (exponentiation)
-    str a, num b: every bth character of a
-    num a, str b: every ath character of b
+    str a, num b: append a[0] until a is length b (spaces are used if a is empty)
```

Comment:
Wait, isn't this change already done in the other PR?

Reply:
yes, but it's the exact same.